lib/tasks/evm.rake

```diff
@@ -96,7 +96,13 @@ namespace :evm do
       "redeployment"   => 5,
       "upgrade"        => 6
     }
+
     status = EvmApplication.deployment_status
+    if status == "new_replica"
+      puts "Database contains the following server records:"
+      MiqServer.all.each { |server| puts "  #{server.guid}: id: #{server.id}, name: #{server.name}, zone: #{server.zone.description}" }
```

Comment:
@bdunne gah, right after I merged, I realized that if we have a server in a broken state, then zone could be nil...it's super rare, but maybe we should be defensive here?  Up to you.

Reply:
https://github.com/ManageIQ/manageiq/pull/21201